azerothcore/azerothcore-wotlk
# fix(Core/SAI): Enable SMART_ACTION_GAME_EVENT_STOP/START

SMART_ACTION_GAME_EVENT_START and SMART_ACTION_GAME_EVENT_STOP do not
function properly.
- Removed return false; at top of both case statements.
- Tested working in my environment.





## Changes Proposed:
-  Remove return false; on line 1062 and 1082 in the SmartScriptMgr.cpp file
-  


## Issues Addressed:
SMART_ACTION_GAME_EVENT_START and SMART_ACTION_GAME_EVENT_STOP do not function properly.




## Tests Performed:

-Built without errors.
-Tested fix on my current environment: AzerothCore rev. 3d60ce8e6e3f+ 2021-01-24 21:21:48 -0500 (master branch)


## How to Test the Changes:

- Add a smart script using action_type 112 SMART_ACTION_GAME_EVENT_START and/or action_type 111 SMART_ACTION_GAME_EVENT_STOP.
- The Event should start or stop when smart script is acitvated.  

## Known Issues and TODO List:



## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
